src/CNN/networks/Training.py

```diff
@@ -144,7 +145,7 @@ def benchmark(self, n_datapoints, datapath, rotations=True, model=None, ensemble
                 for j in range(24):
                     with h5py.File(datafile, 'r') as file:
                         data = rot.rotation(data=file[str(i) + '/data'][()][0], k=j)
-                        label = -np.log10(np.exp(-(file[str(i) + '/label'][()])))
```

Comment:
Thanks. I had to add this, because I had to change the natural log to log10, because the old code used log2 instead of log10.